components/breadcrumb/css/_mixin.scss

```diff
@@ -42,70 +48,20 @@
   padding-block-start: var(--utrecht-breadcrumb-item-padding-block-start, 8px);
   padding-inline-end: var(--utrecht-breadcrumb-item-padding-inline-end, 8px);
   padding-inline-start: var(--utrecht-breadcrumb-item-padding-inline-start, 8px);
-}
-
-/* stylelint-disable-next-line block-no-empty */
-@mixin utrecht-breadcrumb__text {
-}
-
-.utrecht-breadcrumb--arrows {
-  /* https://css-tricks.com/triangle-breadcrumbs/ */
-  --utrecht-breadcrumb-arrow-size: 24px;
-
-  overflow: hidden;
-
-  .utrecht-breadcrumb__link {
-    padding-inline-end: 0;
-    position: relative;
+  text-decoration: var(--utrecht-breadcrumb-link-text-decoration);
+  &:hover {
+    text-decoration: var(--utrecht-breadcrumb-link-hover-text-decoration);
+    text-decoration-thickness: var(--utrecht-breadcrumb-link-hover-text-decoration-thickness);
```

Comment:
This change breaks the existing hover style of Utrecht, the thick underline.

In addition to the CSS changes, it would be great if you would include stories for that simulate these hover states, so we have visual regression tests to prevent this situation in the future.